gabbi/tests/test_fixtures.py

```diff
@@ -13,10 +13,9 @@
 """Use mocks to confirm that fixtures operate as context managers.
 """
 
+import mock
```

Comment:
test failures suggest using this instead:

```suggestion
from unittest import mock
```